Source/WebCore/crypto/cocoa/SerializedCryptoKeyWrapMac.mm

```diff
@@ -211,6 +211,8 @@ bool deleteDefaultWebCryptoMasterKey()
 
 bool wrapSerializedCryptoKey(const Vector<uint8_t>& masterKey, const Vector<uint8_t>& key, Vector<uint8_t>& result)
 {
+    if (masterKey.isEmpty())
```

Comment:
Does the code forbid using an empty master key before? If not, I wonder whether this could cause breakage on existing data.
For this function itself, it seems it does not prevent using an empty key.

Reply:
This check is failfast. If the masterKey is zero length there is no encryption to be done. So I have added that. The underlying crypto library will return the same error.